server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryErrorTest.java

```diff
@@ -141,29 +156,31 @@ public void setUp() throws Exception {
             .addDomain(DEFAULT_DOMAIN)
             .addUser(FROM, PASSWORD);
 
-        requestSpecificationForMockSMTP1 = requestSpecification(mockSmtp);
-        requestSpecificationForMockSMTP2 = requestSpecification(mockSmtp2);
-        RestAssured.requestSpecification = requestSpecificationForMockSMTP1;
-        RestAssured.enableLoggingOfRequestAndResponseIfValidationFails();
+        mockSMTP1Configuration = configurationClient(mockSmtp);
+        mockSMTP2Configuration = configurationClient(mockSmtp2);
     }
 
     @After
     public void tearDown() {
         jamesServer.shutdown();
-        with().delete("/smtpMails");
+
+        mockSMTP1Configuration.clearBehaviors();
```

Comment:
I don't like to have to know about what I need to cleanup, can't we have a single method that reset the content of the server?